src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -33,7 +33,7 @@ import createCustomModalStackNavigator from './createCustomModalStackNavigator';
 import Permissions from '../../Permissions';
 
 // Main drawer navigator
-import MainDrawerNavigator from './MainDrawerNavigator';
+import MainDrawerNavigator from './mainDrawerNavigator';
```

Comment:
Not sure why this filename changed to a lower case, but probably a misunderstanding as it should be `PascalCase`